src/pages/organize/[orgId]/people/index.tsx

```diff
@@ -40,21 +36,12 @@ type PeopleViewsPageProps = {
 };
 
 const PeopleViewsPage: PageWithLayout<PeopleViewsPageProps> = ({ orgId }) => {
-  const messages = useMessages(messageIds);
-
   const onServer = useServerSide();
   if (onServer) {
     return null;
   }
 
-  return (
-    <>
-      <Head>
-        <title>{messages.browserLayout.title()}</title>
-      </Head>
```

Comment:
I believe it is a mistake to delete this, because it means that there is no longer any title in the HTML. See screenshot below of how Chrome renders the page before (left) and after (right).

![image](https://github.com/zetkin/app.zetkin.org/assets/550212/4cc6f247-82d7-4458-9a8c-9738206954cd)